README.md

```diff
@@ -56,6 +56,14 @@ To automatically set that, the above example uses the context variable `${{ gith
 
 - `platforms` (list, optional) List of target platform for build.
 
+- `target` (string, optional) the name of a build stage in the Dockerfile to target for the image. This allows multiple images
+  built from a single Dockerfile, e.g., "runtime-A" and "runtime-B".
+
+- `build-args` (string, optional) A list of build-arguments as newline-delimited `arg=value` string pairs. These may be
+  specified in the Dockerfile as `ARG` statements.
```

Comment:
Nit: DM documentation style is to not wrap in the middle of sentences. For formats that do the right thing, we use a newline after each sentence. I've had GitHub interpret those as hard breaks in Markdown, so for GitHub Markdown files, I generally just put the whole paragraph on one line.